bc_obps/service/facility_service.py

```diff
@@ -166,7 +167,7 @@ def list_facilities(
     def get_if_authorized(cls, user_guid: UUID, facility_id: UUID) -> Facility:
         """Retrieve a facility if the user is authorized to access it."""
         facility: Facility = FacilityDataAccessService.get_by_id(facility_id)
-        owner: Operation = facility.current_owner
+        owner: Operation = facility.current_designed_operation
```

Comment:
please rename to `current_designated_operation`

Reply:
:scream: